Reduce sensitivity of the maximum errors. ImageMagick insists, these can
vary between platforms:

http://www.imagemagick.org/discourse-server/viewtopic.php?f=3&t=9766

--- PerlMagick/t/filter.t	2007-09-21 10:42:06.000000000 -0400
+++ PerlMagick/t/filter.t	2007-09-25 14:55:22.000000000 -0400
@@ -101,5 +101,5 @@
 ++$test;
 
-testFilterCompare('input.miff',  q//, 'reference/filter/GaussianBlur.miff', 'GaussianBlur', q/'0.0x1.5'/, 0.02, 0.02);
+testFilterCompare('input.miff',  q//, 'reference/filter/GaussianBlur.miff', 'GaussianBlur', q/'0.0x1.5'/, 0.02, 0.2);
 ++$test;
 
@@ -122,5 +122,5 @@
 ++$test;
 
-testFilterCompare('input.miff',  q//, 'reference/filter/Modulate.miff', 'Modulate', q/brightness=>110,saturation=>110,hue=>110/, 0.05, 0.05);
+testFilterCompare('input.miff',  q//, 'reference/filter/Modulate.miff', 'Modulate', q/brightness=>110,saturation=>110,hue=>110/, 0.05, 0.4);
 ++$test;
 
